src/shared/store/actions/nexus/resource.ts

```diff
@@ -73,8 +74,15 @@ export const fetchAndAssignResource: ActionCreator<ThunkAction> = (
   > => {
     dispatch(fetchResourceAction());
     try {
-      const project: Project = await Project.get(orgLabel, projectLabel);
-      const resource = await project.getResource(resourceId);
+      const resource = await Resource.get(
+        orgLabel,
+        projectLabel,
+        '_',
```

Comment:
Implicit magic value, should be a constant or come from a config

Reply:
fixed in e20dbed86139332f97391b9ce31b78f434665d2d